internal/telemetry/internal/transport/origin.go

```diff
@@ -16,4 +16,5 @@ const (
 	OriginRemoteConfig        Origin = "remote_config"
 	OriginLocalStableConfig   Origin = "local_stable_config"
 	OriginManagedStableConfig Origin = "fleet_stable_config"
+	OriginCalculated          Origin = "calculated"
```

Comment:
This origin is needed by the migration PRs that are based on this branch.

The "calculated" origin is already used by other dd-trace libs such as DotNet, Node and Java. It means the configuration value was automatically determined by our library based for a given environment; we "calculated" it based on various factors